main/webapp/modules/core/scripts/views/data-table/cell-ui.js

```diff
@@ -62,43 +62,50 @@ DataTableCellUI.prototype._render = function() {
   var self = this;
   var cell = this._cell;
 
-  var divContent = $('<div/>')
-  .addClass("data-table-cell-content");
+  var divContent = document.createElement('div');
+  divContent.className = 'data-table-cell-content';
 
-  var editLink = $('<a href="javascript:{}">&nbsp;</a>')
-  .addClass("data-table-cell-edit")
-  .attr("title", $.i18n('core-views/edit-cell'))
-  .appendTo(divContent)
-  .click(function() { self._startEdit(this); });
+  var editLink = document.createElement('a');
+  editLink.className = 'data-table-cell-edit';
+  editLink.setAttribute('title', $.i18n('core-views/edit-cell'));
+  editLink.href = 'javascript:{}';
+  divContent.appendChild(editLink).appendChild(document.createTextNode('\u00A0'));
+  editLink.addEventListener('click', function() { self._startEdit(this); });
 
   $(this._td).empty()
   .unbind()
-  .mouseenter(function() { editLink.css("visibility", "visible"); })
-  .mouseleave(function() { editLink.css("visibility", "hidden"); });
+  .mouseenter(function() { editLink.style.visibility = "visible" })
```

Comment:
That should be possible, I will add it.

Reply:
So I checked it, and passing arguments to an event handler is not possible unless we create a function to do it, which would make it's removal redundant. The other possible way would be to use `event.currentTarget` instead of passing `editLink`, or declaring it as a global variable. But since it's jQuery the event is a jQuery object, and we'd end up using jQuery to style the element.

Reply:
OK, thanks for investigating. Sorry to send you on a wild goose chase!